server/migrations/20210730210120-add-fileOperations.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable("file_operations",{
+      id: {
+        type: Sequelize.UUID,
+        allowNull: false,
+        primaryKey: true,
+      },
+      state: {
+        type: Sequelize.ENUM("creating", "uploading", "complete", "error","expired"),
+        allowNull: false,
+      },
+      type: {
+        type: Sequelize.ENUM("import", "export"),
+        allowNull: false,
+      },
+      key: {
+        type: Sequelize.STRING,
+      },
+      url: {
+        type: Sequelize.STRING,
+      },
+      size: {
+        type: Sequelize.BIGINT,
+        allowNull: false,
+      },
+      userId: {
+        type: Sequelize.UUID,       
+        allowNull: false,
+        references: {
+          model: "users"
+        }
+      },
+      collectionId: {
+        type: Sequelize.UUID,       
+        references: {
+          model: "collections"
+        }
+      },
+      teamId: {
+        type: Sequelize.UUID,
+        allowNull: false,
+        references: {
+          model: "teams"
+        }
+      },
+      createdAt: {
+        type: Sequelize.DATE,
+        allowNull: false,
+      },
+      updatedAt: {
+        type: Sequelize.DATE,
+        allowNull: false,
+      },
+    })
+  },
```

Comment:
We'll need to an index on the `state` field here as it's queried in `utils.gc`

Reply:
Added the index on `state` and `type` because we use both the field while querying.